worker/categorize.py

```diff
@@ -403,11 +404,18 @@ def _is_viable_candidate(self, phrase):
                 return False
 
             titled_words = map(lambda x: x.istitle(), words)
-            if titled_words not in [[True, False, True], [True, True,True]]:
+            if titled_words not in [[True, False, True], [True, True, True]]:
                 return False
 
         return True
 
+    def _is_nested(self, keyword, other_keywords):
+        for other_keyword in other_keywords:
+            if keyword != other_keyword and keyword in other_keyword:
```

Comment:
That's just how searching works. I don't know if it needs a comment for that particular line, but I can document the function.